web/themes/custom/vote_gov/templates/node/node--page--full.html.twig

```diff
@@ -0,0 +1,105 @@
+ *   classes:
+ *   - node: The current template type (also known as a "theming hook").
+ *   - node--type-[type]: The current node type. For example, if the node is an
+ *     "Article" it would result in "node--type-article". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ */
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+] %}
+
+<div class="bg-lightblue">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+      </div>
+      <div class="tablet:grid-col-8 right-shift">
+        <section class="voter-info page-type--basic">
+          <div class="about-page">
+            {# Title banner and main content #}
+            {{ content }}
+            {# Homepage button #}
+            <a href="/">
+              <button type="button" class="usa-button">
+              Get Registered
+              </button>
+            </a>
```

Comment:
Remove the link/button from the template. We will need to add this in the wysiwyg content rather than in the template.

In the Basic HTML text format located here: http://vote-gov.lndo.site/admin/config/content/formats/manage/basic_html

You will need to add the 'Style' component to the wysiwyg toolbar

Then add a new style for `a.usa-button|Button styled link`

We will need to do some minor style adjustments as needed to make it match the defauly button style.

Reply:
I tried to add it as `button.usa-button|Button styled link`, but that would required a plugin for the styles to be able to handle the <button>. Is this something we should do?
If not:
Adding it as a link (like you did above) would require that I override a link style that makes the text blue instead of white. Should I be doing this in field.html.twig?

Reply:
Use `a.usa-button` We will use css to override the style in the `vote_gov/src/sass/uswds-overrides/_override-usa-button.scss`